subworkflows/nf-core/vcf_phase_shapeit5/main.nf

```diff
@@ -0,0 +1,81 @@
+include { BEDTOOLS_MAKEWINDOWS              } from '../../../modules/nf-core/bedtools/makewindows/main.nf'
+include { SHAPEIT5_PHASECOMMON              } from '../../../modules/nf-core/shapeit5/phasecommon/main'
+include { SHAPEIT5_LIGATE                   } from '../../../modules/nf-core/shapeit5/ligate/main'
+include { BCFTOOLS_INDEX as VCF_INDEX1      } from '../../../modules/nf-core/bcftools/index/main.nf'
+include { BCFTOOLS_INDEX as VCF_INDEX2      } from '../../../modules/nf-core/bcftools/index/main.nf'
+
+workflow VCF_PHASE_SHAPEIT5 {
+
+    take:
+    ch_vcf        // channel (mandatory): [ [id, ref], vcf, csi, pedigree, region ]
```

Comment:
```suggestion
    ch_vcf        // channel (mandatory): [ val(meta), path(vcf), path(csi), path(pedigree), val(region) ]
```
It's preffered to specify whether the variable is a value or path in this way. This makes it really clear to the end users what to do. Can you change all comments at the input and output to this format. Can you also change the structure in the meta.yml to this format?